docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -10,225 +10,77 @@ images: ["/installation/thumbnails/esp32-espressif.png"]
-
-```sh { class="command-line" data-prompt="$"}
-sudo apt-get install git wget flex bison gperf python3 python3-pip python3-venv cmake ninja-build ccache libffi-dev libssl-dev dfu-util libusb-1.0-0
-```
-
-{{% /tab %}}
-{{% tab name="macOS" %}}
-
-```sh { class="command-line" data-prompt="$"}
-brew install cmake ninja dfu-util
-```
-
-{{% /tab %}}
-{{% /tabs %}}
+## Install the Micro-RDK
 
-#### Install and Activate the ESP-IDF
+The [Micro-RDK Installer](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer) is a CLI that allows you to flash a build of Micro-RDK, along with your robot's credentials and your wifi information, directly to your ESP32.
 
-Clone Viam's fork of the ESP-IDF, the development framework for Espressif SoCs (System-on-Chips) supported on Windows, Linux and macOS:
+With this installation, you can use your ESP32 with all supported resource APIs, but you cannot write your own code directly interacting with the chip.
+If you want to program the chip directly, follow the setup instructions in [the Micro-RDK Development Setup](/installation/prepare/microcontrollers/development-setup/) instead.
 
-```sh { class="command-line" data-prompt="$"}
-mkdir -p ~/esp
-cd ~/esp
-git clone --depth 1 -b v4.4.4 --single-branch --recurse-submodules --shallow-submodules https://github.com/npmenard/esp-idf
-```
+### Download the Micro-RDK Installer
 
-Then, install the required tools for ESP-IDF:
+Select the appropriate pre-built binary for the architecture of your development machine:
 
-```sh { class="command-line" data-prompt="$"}
-cd ~/esp/esp-idf
-./install.sh esp32
-```
+- [Linux (x86_64)](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-amd64-linux)
+- [Linux (Aarch64)](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-arm64-linux)
+- [MacOS](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-macos)
+- [Windows](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-windows.exe)
 
-To activate ESP-IDF, run the following command to source (`.`) the activation script `export.sh`:
+### Flash your ESP32 with the Micro RDK Installer
 
-```sh { class="command-line" data-prompt="$"}
-. $HOME/esp/esp-idf/export.sh
-```
+Navigate to [the Viam app](https://app.viam.com) and [add a new robot](/manage/fleet/robots/#add-a-new-robot) in your desired location.
 
-{{< alert title="Tip" color="tip" >}}
-To avoid conflicts with other toolchains, adding this command to your `.bashrc` or `.zshrc` is not recommended.
-Instead, save this command to run in any future terminal session where you need to activate the ESP-IDF development framework.
-{{< /alert >}}
+1. Click on the name of the robot to go to the robot's page.
+2. Click on the **Setup** tab.
+3. Regardless of your operating system, select **Mac** and press the button that appears in Step 2 to download the Viam app config for your robot.
```

Comment:
Q:  Could we change these instructions once https://viam.atlassian.net/browse/RSDK-5080 is merged?

Reply:
Can do, I made a ticket to track: https://viam.atlassian.net/jira/software/c/projects/DOCS/boards/31?quickFilter=41&selectedIssue=DOCS-1273